subprojects/docs/src/snippets/java/customDirs/kotlin/build.gradle.kts

```diff
@@ -55,9 +55,17 @@ reporting.baseDir = file("my-reports")
 project.setProperty("testResultsDirName", "$buildDir/my-test-results")
 
 tasks.register("showDirs") {
+    val rootDirPath = project.rootDir.toPath()
+
+    val reportsDir: Directory = project.the<ReportingExtension>().getBaseDirectory().get()
+    val reportsDirPath = reportsDir.getAsFile().toPath()
+
+    val testResultsDir: Directory = project.the<JavaPluginExtension>().getTestResultsDir().get()
```

Comment:
🔴 The `project.java` accessor should be available at this point. Is it?

Reply:
updated PR to use `project.java` here